src/Peachpie.Library/Hash.cs

```diff
@@ -2973,12 +2973,20 @@ public static PhpArray password_get_info(string hash)
         /// One-way string hashing.
         /// </summary>
         /// <returns>Returns the hashed string or a string that is shorter than 13 characters and is guaranteed to differ from the salt on failure.</returns>
-        public static string crypt(string str, string salt/* mandatory since 5.6 */)
+        public static string crypt(Context ctxPchp, string str, string salt/* mandatory since 5.6 */)
```

Comment:
`Context ctxPchp` is not needed.

Reply:
Resolved https://github.com/peachpiecompiler/peachpie/pull/593/commits/6f1865f58f6166fda85c7352d98cab2d049e5ddf